rts/Lua/LuaHandle.h

```diff
@@ -326,6 +328,9 @@ class CLuaHandle : public CEventClient
 
 		std::string killMsg;
 
+		std::map <int, std::vector <std::pair <int, std::vector <int>>>> delayedCallsByFrame;
```

Comment:
I am not entirely happy with this type, but I don't think introducing a typedef or something like that would help since it isn't actually directly referenced anywhere else (thanks to `auto`).